code/modules/mob/living/simple_animal/vore/otie.dm

```diff
@@ -128,33 +128,37 @@
 		else if(tamed == 1 && isrobot(found_atom))
 			return null
 		else
+			if(resting)
+				lay_down()
 			return found_atom
 	else
 		return null
 
 /mob/living/simple_animal/otie/friendly/security/Found(var/atom/found_atom)
```

Comment:
The Found() verb here still doesn't use any of the default functionality with mobs, and I feel like there's a great deal of redundant code here that I don't like.

Same goes for regular oties too.

as far as I can tell the only thing they do differently is eat mice and lay down.

Reply:
The found thing does not have any default functionality.
![kuva](https://user-images.githubusercontent.com/13697337/31151422-0dc4a52e-a8a0-11e7-93ad-57c321f8a212.png)

Reply:
And the lay down thing there happens to be the exact fix for the idle resting issue during combat and shit. Notice a slappee, haul up that ass and go slappin. The if resting -> lay down (lay_down is a toggle fyi) basically is a stand up if already resting.

Reply:
Anyway, feel free to mark all that redundant shit for me so I can either trim it or give explanations as the comments above.

Reply:
Oh nevermind then.